src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -1028,8 +1112,8 @@ public static void createSinkBQExistingDatatypeTable() throws IOException, Inter
       BeforeActions.scenario.write("Error inserting the record in the table" + e.getStackTrace());
     }
 
-    PluginPropertyUtils.addPluginProp(" bqTargetTable",  bqTargetTable);
-    BeforeActions.scenario.write("BQ Target Table " +  bqTargetTable + " updated successfully");
+    PluginPropertyUtils.addPluginProp(" bqTargetTable", bqTargetTable);
+    BeforeActions.scenario.write("BQ Target Table " + bqTargetTable + " updated successfully");
   }
```

Comment:
Revert all the above changes which are highlighted because of indentation.

Reply:
done.